src/components/FormattedNumber/FormattedNumber.jsx

```diff
@@ -29,13 +46,26 @@ const FormattedNumber = forwardRef(
     }, [value, decimalPrecision, local, compact]);
 
     if (validateValue(value)) {
-      return <span className={`${baseClassName}__place-holder`}>{emptyPlaceHolder}</span>;
+      return (
+        <span
+          className={cx(`${CSS_BASE_CLASS}__place-holder`)}
```

Comment:
no css modules class here?

Reply:
That's correct. There is no such class in the stylesheet - so it's being used only for overrides